users/views.py

```diff
@@ -1,3 +1,55 @@
-from django.shortcuts import render
+import json, jwt, requests
+
+
+from django.views import View
+from django.http  import JsonResponse
+
+from users.models import User, UserLevel
+from my_settings  import SECRET_KEY, REST_API_KEY
+
+class KaKaoSignInView(View):
+    def post(self, request):
+        try:
+            REDIRECT_URI = "http://localhost:3000/users/kakao"
+            POINT        = 1000000
+            
+            data = json.loads(request.body)
+            code = data['code']
+
+            kakao_token_request = requests.get(    
+                f"https://kauth.kakao.com/oauth/token?grant_type=authorization_code&client_id={REST_API_KEY}&redirect_uri={REDIRECT_URI}&code={code}",
+            )
+            kakao_json = kakao_token_request.json()
+            profile_request = requests.get(    
+                "https://kapi.kakao.com/v2/user/me",
+                headers = {"Authorization" : f"Bearer {kakao_json['access_token']}"}
+            )
```

Comment:
변수에 담기는 값을 명확하게 작성해주세요~
requests.get() 호출시 요청에 대한 응답이 반환되므로 `response` 변수에 담아주시는게 좋습니다.

```suggestion
            response = requests.get(    
                "https://kapi.kakao.com/v2/user/me",
                headers = {"Authorization" : f"Bearer {kakao_json['access_token']}"}
            )
```